zbus/Cargo.toml

```diff
@@ -108,7 +108,7 @@ nix = { version = "0.27", default-features = false, features = [
 [target.'cfg(target_os = "macos")'.dependencies]
 # FIXME: This should only be enabled if async-io feature is enabled but currently
 # Cargo doesn't provide a way to do that for only specific target OS: https://github.com/rust-lang/cargo/issues/1197.
-async-process = "1.7.0"
+async-process = "2.0.0"
```

Comment:
Did not and can not test this change.

Reply:
Me neither. :) I think some tests cover this so CI will tell us.